bedrock/base/templatetags/helpers.py

```diff
@@ -15,8 +15,12 @@
 
 
 @library.global_function
-def send_to_device_countries():
-    return '|%s|' % '|'.join(cc.lower() for cc in settings.SEND_TO_DEVICE_COUNTRIES)
+def send_to_device_countries(message_set):
```

Comment:
As the logic is now getting a bit complex between locales and countries, email and SMS, I wonder if we should rename this to `send_to_device_sms_countries`?

I don't want to blow up the scope of this PR, but there are a few other places we could make this distinction a bit clearer as well. Maybe a follow-up PR.